<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMSExp</string>
  </property>
  <widget class="gui::widgets::ButtonSelecterWidget" name="m_central_widget"/>
  <widget class="QStatusBar" name="m_status_bar"/>
  <widget class="QToolBar" name="m_tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_action_quit"/>
   <addaction name="separator"/>
   <addaction name="m_action_reset"/>
  </widget>
  <action name="m_action_quit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitte l'application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_action_reset">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Réinitialiser</string>
   </property>
   <property name="toolTip">
    <string>Réinitialiser les fichiers chargés.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::widgets::ButtonSelecterWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/ButtonSelecterWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
